examples/py_proto_library/WORKSPACE

```diff
@@ -0,0 +1,24 @@
+workspace(name = "rules_python_py_proto_library_example")
```

Comment:
Would this example be cleaner/simpler if we used bzlmod?

Reply:
I think we should make every example "hybrid" with both workspace and bzlmod usage. we've been migrating all the Aspect rulesets to work this way. Keeps us honest that the two are feature-parity

Reply:
https://github.com/aspect-build/rules_swc/commit/af52471ac451d647fd187b9e92162d6a6199e700 is an example

Reply:
Added bzlmod support for the example.